manifest-list.md

```diff
@@ -11,7 +11,8 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 - **`schemaVersion`** *int*
 
   This REQUIRED property specifies the image manifest schema version.
-  For this version of the specification, this MUST be `2` to ensure backward compatibility with older versions of Docker. The value of this field will not change. This field MAY be removed in a future version of the specification.
+  For this version of the specification, this MUST be `2` to ensure backward compatibility with older versions of Docker.
+  The value of this field will not change. This field MAY be removed in a future version of the specification.
```

Comment:
If you're trying to break this into one sentence per line, you'll want to put another newline in between “change.” and “This”.  But you can probably make this PR without touching the `schemaVersion` definition at all.